src/components/layout/tables/AntRow/AntRow.tsx

```diff
@@ -75,6 +81,13 @@ function AntRow({
               }
             : {}
         }
+        onClick={
+          isMobile
+            ? () => setShowManageModal(true)
+            : () => {
+                return;
+              }
```

Comment:
not showing modal on mobile?

Reply:
backwards actually, if its mobile the row itself is the button that pops the modal, if not mobile it is not a button and the button to pop the modal is the manage button from the manage assets buttons

Reply:
ah misread - some small items on mobile look crunched

<img width="382" alt="image" src="https://user-images.githubusercontent.com/12800001/214074288-82055d11-a7fb-4c6f-8fb3-60d7fc9ad96e.png">

Reply:
i also feel like a magnifying glass might be better for viewing the ant details on mobile than clicking the status icon. it's unclear that that will show this page ☝️ - what do you think?

Reply:
Ok yeah I need to take a look at some text alignment it seems. Hm, I dislike using the search icon for something that isnt running a search, what about this? 
![image](https://user-images.githubusercontent.com/85306700/214163199-8062fd11-4adf-4f7e-900b-b79bd5b85d48.png)

Reply:
or maybe this (standardish fullscreen icon) 
![image](https://user-images.githubusercontent.com/85306700/214164051-f9a50f60-1dc1-4376-8dd1-826c2b39994d.png)